operators/midas_op.py

```diff
@@ -0,0 +1,92 @@
+import os
+from typing import Callable
+
+import cv2
+import numpy as np
+import torch
+from dora import DoraStatus
+
+IMAGE_WIDTH = 1920
+IMAGE_HEIGHT = 1080
+DEVICE = os.environ.get("PYTORCH_DEVICE") or "cpu"
+MIDAS_PATH = os.environ.get("MIDAS_PATH")
+MIDAS_WEIGHT_PATH = os.environ.get("MIDAS_WEIGHT_PATH")
+MODEL_TYPE = os.environ.get("MODEL_TYPE")
+MODEL_NAME = os.environ.get("MODEL_NAME")
+
+
+# example source: https://pytorch.org/hub/intelisl_midas_v2/
+class Operator:
+    def __init__(self):
+        if MIDAS_PATH is None:
+            # With internet
+            self.model = torch.hub.load(
+                "intel-isl/MiDaS",
+                MODEL_TYPE,
+            )
+        else:
+            # Without internet
+            self.model = torch.hub.load(
+                repo_or_dir=MIDAS_PATH,
+                model=MODEL_NAME,
+                weights=MIDAS_WEIGHT_PATH,
+                source="local",
+            )
+        midas_transforms = torch.hub.load("intel-isl/MiDaS", "transforms")
```

Comment:
```suggestion
        if MIDAS_PATH is None:
            # With internet
            self.model = torch.hub.load(
                "intel-isl/MiDaS",
                MODEL_TYPE,
            )
            midas_transforms = torch.hub.load("intel-isl/MiDaS", "transforms")
        else:
            # Without internet
            self.model = torch.hub.load(
                repo_or_dir=MIDAS_PATH,
                model=MODEL_NAME,
                weights=MIDAS_WEIGHT_PATH,
                source="local",
            )
            midas_transforms = torch.hub.load(
                repo_or_dir=MIDAS_PATH, model="transforms", source="local"
            )
 
```

By adding `repo_or_dir` and `source=local` in the `torch.hub.load`, you can make midas transform offline.